### DISCUSSION TITLE
Can't get the post :File to work

### DISCUSSION THREAD

**Discussion by donbenci13** [swellstores/community#573] [Help] [answered]
Hi, I'm trying the create file but it just doesn't work for me.

images get uploaded but they are always broken (https://cdn.swell.store/pili_test/66f5f3533784a10012f8fb3b/767c01de1d34e5a2c650f81b4d63879f/someimage.jpeg). below is a the key test payload I use with Postman.
[testing file create.txt](https://github.com/user-attachments/files/17157192/testing.file.create.txt)

<img width="933" alt="image" src="https://github.com/user-attachments/assets/ed6796ba-c888-426d-b025-bd0620f1ba72">

the image I used here is an image from another post I saw here while looking for answers.
![75](https://github.com/user-attachments/assets/ba280777-6b33-4367-b026-e972dfd68240)

**Comment by logeshswell** [comment, marked as answer]
Can you follow the instructions from this comment?
https://github.com/orgs/swellstores/discussions/3#discussioncomment-4242550

Basically you need to replace `$base64` with `$binary` and it should work normally.

**Reply by donbenci13** [reply]
it actually worked with both `$binary` and `$base64` but I had to cut off the `data:image.+;base64` from the string.

I had a rough time making it work with the API call but works perfectly with the swell-node.

thanks!
